### PR TITLE
Add WeAct C6

### PR DESCRIPTION
**Pull request by bill88t**
Creator ID PR pending.
https://github.com/creationid/creators/pull/46

### CODE REVIEW

File: tools/ci_check_duplicate_usb_vid_pid.py [hunk 1]
```diff
@@ -63,6 +63,7 @@
         "espressif_esp32s2_devkitc_1_n8r2",
     ],
     "0x239A:0x102E": ["weact_studio_pico", "weact_studio_pico_16mb"],
+    "0x00111011:0x00C60001": ["weact_esp32c6_n4", "weact_esp32c6_n8"],
```
Comment (tannewt) [inline tools/ci_check_duplicate_usb_vid_pid.py:66]: Let's just give two different creation ids for these. We have plenty of numbers. Maybe the last digit could be flash size?

**Review by tannewt — CHANGES_REQUESTED** [review]

File: tools/ci_check_duplicate_usb_vid_pid.py (hunk 1 above)
Comment (bill88t) [reply tools/ci_check_duplicate_usb_vid_pid.py]: Alright, changed, as specified in the Creators ID pr. This entry in the script was removed.

File: ports/espressif/boards/weact_esp32c6_n4/mpconfigboard.mk [hunk 2]
```diff
@@ -0,0 +1,11 @@
+CIRCUITPY_CREATOR_ID = 0x00111011
```
Comment (tannewt) [inline ports/espressif/boards/weact_esp32c6_n4/mpconfigboard.mk:1]:
```suggestion
CIRCUITPY_CREATOR_ID = 0x10111011
```

File: ports/espressif/boards/weact_esp32c6_n8/mpconfigboard.mk [hunk 3]
```diff
@@ -0,0 +1,11 @@
+CIRCUITPY_CREATOR_ID = 0x00111011
```
Comment (tannewt) [inline ports/espressif/boards/weact_esp32c6_n8/mpconfigboard.mk:1]:
```suggestion
CIRCUITPY_CREATOR_ID = 0x10111011
```

**Review by tannewt — CHANGES_REQUESTED** [review]
Two small creator id updates to match the creators repo.

**Comment by bill88t** [comment]
Whoops. Thanks for catching that!
Co-commited the suggested changes.

**Review by tannewt — APPROVED** [review]
Thank you!

**Pull request merged**